azurerm_api_management/output.tf

```diff
@@ -1,4 +1,5 @@
-output "id" {
-  description = "The Azure resource group id"
-  value       = "${module.azurerm_api_management.id}"
-}
+# output "id" {
```

Comment:
please, remove commented output if not needed. Put a comment instead...or add relevant output (maybe at least the name of the api manager created?)